easybuild/tools/modules.py

```diff
@@ -588,7 +588,7 @@ def modpath_extensions_for(self, mod_names):
         modpath_exts = {}
         for mod_name in mod_names:
             modtxt = self.read_module_file(mod_name)
-            useregex = re.compile(r"^\s*module\s+use\s+(\S+)", re.M)
+            useregex = re.compile(r'^\s*module\s+use\s+"?([^" \t\n\r\f\v]+)"?', re.M)
```

Comment:
@geimer: please use `\s` instead of ` \t\n\r\f\v`, cfr. https://docs.python.org/2/library/re.html

```
\s
When the UNICODE flag is not specified, it matches any whitespace character,
this is equivalent to the set [ \t\n\r\f\v]. 
```